osu.Game.Rulesets.Osu/Mods/OsuModFlashlight.cs

```diff
@@ -32,8 +32,8 @@ public class OsuModFlashlight : ModFlashlight<OsuHitObject>, IApplicableToDrawab
             Precision = default_follow_delay,
         };
 
-        [SettingSource("Flashlight size", "Multiplier applied to the default flashlight size.")]
-        public override BindableFloat SizeMultiplier { get; } = new BindableFloat
+        [SettingSource("Starting flashlight size", "Multiplier applied to the default flashlight size.")]
```

Comment:
Why is `SettingSource` specified here? Should not be required.

Reply:
It was being specified before this pull request though